authentication.go

```diff
@@ -105,5 +106,14 @@ func (a *App) ValidateAuthToken(authToken string) (string, bool) {
 		return "", false
 	}
 
+	audience, ok := claims["aud"].([]string)
+	if !ok {
+		return "", false
+	}
+
+	if !slices.Contains(audience, a.ID) {
```

Comment:
our library offers a comparator func for this, which also takes care of the `string` vs `[]string` type check: `claims.VerifyAudience`

Reply:
Change this check to use our library comparator function